pkg/cli/root.go

```diff
@@ -31,6 +31,9 @@ var rootCmd = func() cli.Command {
 			"validate": func() cli.Command {
 				return &ValidateCommand{}
 			},
+			"pull": func() cli.Command {
+				return &ValidatePullCommand{}
```

Comment:
Should this just be `PullCommand`?

I wonder if "tail" might be a better name (referring to the Linux command `tail`).

Reply:
It does both things pull and validate, I am ok with `PullCommand` since lumberctl is a validation tool, it should imply it will also run validation. 

If we should use `tail`, it'd be better to rename pull everywhere? What will be a good replacement for `LogPuller`?

Reply:
Maybe tail is a bad name then :)

Reply:
Is `LogTailer` a good name? I am not native english speaker, can't tell...

Reply:
Switched to tail, but kept logPuller. Let me know if it is confusing. Thanks!

Reply:
Resolving, feel free to reopen